src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -71,16 +50,17 @@ public String getColor() {
         return color;
     }
 
-    public int getDistance() {
-        return distance;
-    }
+    public void addLineBridge(LineBridge lineBridge) {
+        Objects.requireNonNull(lineBridge, "구간 정보가 필요합니다.");
+        lineBridges.add(lineBridge);
 
-    public Station getDownStation() {
-        return downStation;
+        if (lineBridge.getLine() != this) {
```

Comment:
```suggestion
        if (lineBridge.isSameLine(this)) {
```

단순한 `getter` 보다는 메시지를 던져보는것도 좋을것 같네요!